### PR TITLE
Added unit testing

### PR DESCRIPTION
**Pull request by sobs0**

### CODE REVIEW

**Review by sobs0 — COMMENTED** [review]
Resolved

File: backend/.github/workflows/test.yml
Comment (phtngyn) [inline backend/.github/workflows/test.yml:1]: workflow file must be at root level of project

File: backend/contracts/CommitteeManager.sol
Comment (phtngyn) [inline backend/contracts/CommitteeManager.sol:1]: why there are so many changes? only reorganized code or you also added some logics?

File: backend/contracts/EscrowManager.sol
Comment (phtngyn) [inline backend/contracts/EscrowManager.sol:1]: same as above

File: backend/contracts/ReviewManager.sol
Comment (phtngyn) [inline backend/contracts/ReviewManager.sol:1]: same as above

File: backend/contracts/UserManager.sol
Comment (phtngyn) [inline backend/contracts/UserManager.sol:1]: same as above

File: backend/script/Counter.s.sol
Comment (phtngyn) [inline backend/script/Counter.s.sol:1]: maybe delete this file?

File: backend/.github/workflows/test.yml
Comment (sobs0) [reply backend/.github/workflows/test.yml:1]: How do I do that?

File: backend/contracts/CommitteeManager.sol
Comment (sobs0) [reply backend/contracts/CommitteeManager.sol:1]: I did change some part of functions and restrictions but not changed logic actually, but I also did not reorganize so I don't know why it's marked...

File: backend/script/Counter.s.sol
Comment (sobs0) [reply backend/script/Counter.s.sol:1]: yeah I guess that was installed when i installed the unit testing framework, didn't see it

**Comment by Knorrsche** [comment]
Added Tests for EscrowManager

**Comment by phtngyn** [comment]
I will rewrite tests with hardhat, which is already available for testing

**Comment by phtngyn** [comment]
@sobs0 can you check why `forge build` and `forge test` log `nothing to compile in ci pipeline?

**Comment by phtngyn** [comment]
> @sobs0 can you check why `forge build` and `forge test` log `nothing to compile in ci pipeline?

https://github.com/skillchain-thws/app/actions/runs/7817000754/job/21323862159?pr=2

**Pull request merged**